From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Patrick McCarty <patrick.mccarty@intel.com>
Date: Thu, 17 Dec 2020 20:35:00 +0000
Subject: [PATCH] Remove en_US latin1 test

The localen1 test output currently prints a warning:

  ./localenl.sh: line 39: warning: setlocale: LC_ALL: cannot change
  locale (en_US.ISO-8859-1): No such file or directory

This warning is from bash itself, likely introduced after the bash 5.1
update. But it's a valid warning, because the en_US.ISO-8859-1 (en_US
latin1) locale is not defined in Clear Linux OS.

Not sure how upstream should adjust the test, so for now, keep the patch
distro-specific.

Upstream-Status: Inappropriate
Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
---
 test/localenl.ok | 1 -
 test/localenl.sh | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/test/localenl.ok b/test/localenl.ok
index 47a0748..4e4790e 100644
--- a/test/localenl.ok
+++ b/test/localenl.ok
@@ -1,4 +1,3 @@
 LC_ALL=C passed
 LC_ALL=POSIX passed
-LC_ALL=en_US.ISO-8859-1 passed
 LC_ALL=en_US.UTF-8 passed
diff --git a/test/localenl.sh b/test/localenl.sh
index ca3ee64..a87a58e 100755
--- a/test/localenl.sh
+++ b/test/localenl.sh
@@ -36,7 +36,7 @@
 AWK=${AWK:-../gawk}
 
 # April 2010: Remove UNKNOWN, causes spurious failures on some systems
-for LC_ALL in C POSIX en_US.ISO-8859-1 en_US.UTF-8 #UNKNOWN 
+for LC_ALL in C POSIX en_US.UTF-8 #UNKNOWN 
 do
 export LC_ALL
 cat <<EOF |
